apps/obs-web/src/_store/user.module.ts

```diff
@@ -13,14 +13,23 @@ const actions: ActionTree<UserState, RootState> = {
   setClosedTripsTable({ commit }: any, choice: boolean) {
     commit('setClosedTripsTable', choice);
   },
-  setShowLogbookRetained({ commit}: any, choice: boolean) {
+  setShowLogbookRetained({ commit }: any, choice: boolean) {
     commit('setShowLogbookRetained', choice);
   },
   setAutoHideMenu({ commit }: any, choice: boolean) {
     commit('setAutoHideMenu', choice);
   },
-  setShowOpenEmTrips({ commit}: any, choice: boolean) {
+  setShowOpenEmTrips({ commit }: any, choice: boolean) {
     commit('setShowOpenEmTrips', choice);
+  },
+  setObserverMode({ commit }: any, choice: boolean) {
```

Comment:
What's the difference between userRoles and observer/captain mode? Is setting userRole = ['observer'] equivalent to observerMode = true? If so then do we need both user roles and the booleans representing the different modes?

Reply:
observer mode and captain mode are toggles that allow staff to see what the app would look like if they were logged in as an observer or captain - just easier than creating testing accounts or adding and removing their roles to match what an observer or captain sees.  Only staff roles can see these toggles, and they likely won't be needed once the app is done and in use.